i18n/translations/es.json

```diff
@@ -0,0 +1,3139 @@
+{
+	"_t_min read_t_": "min de lectura",
```

Comment:
when such a file is changed, it will show in crowdin as a new line to proofread/translate, even though we have just changed the id. Can we confirm that we want to do it once for our existing ids and translations?

Reply:
These files won't affect crowdin at all. we just generate ( or transform ) these files based on the translations we download from crowdin. the main purpose of these files is to remove `crc-32` hashing on run time.

Reply:
ok